compatibility-kit/javascript/features/pending/pending.feature

```diff
@@ -0,0 +1,13 @@
+Feature: Pending steps
+
+  During development, step definitions can signal at runtime that they are
+  not yet implemented (or "pending") by returning or throwing a particular
+  value.
+
+  This causes subsequent steps in the scenario to be skipped, and the overall
+  result to be treated as a failure.
+
+  Scenario: Unimplemented step signals pending status
+    Given a step
+    When a step that isnt implemented yet
+    Then a step
```

Comment:
Dropping the other two steps would simply this.

Reply:
I wanted the messages that come out of this to show that the step before runs and passes, and the step after is skipped - I think that's key to properly implementing this?

Reply:
Maybe I should make this explicit in the wording of these steps?

Reply:
> I wanted the messages that come out of this to show that the step before runs and passes, and the step after is skipped - I think that's key to properly implementing this?

I think that should be a separate test case then. Could be much more generic. Any step after a non-passing step is skipped.

Reply:
I kind of prefer to have the test cases for pending etc be fairly complete even if there is a little duplication overall with the skip-subsequent-steps concept, especially as there is some patchiness in our implementations (e.g. ruby doesn't support pending or skipped it seems).

Reply:
I understand, but when the actual and expected result differ it is very hard to to see what the problem is when there is extra fluff laying around. 

Consider instead using multiple examples:

```
  Scenario: Unimplemented step signals pending status
    Given a step that isnt implemented yet

  Scenario: Steps after unimplemented steps are skipped
    Given a step that isnt implemented yet
    Then a skipped step

  Scenario: Steps before unimplemented steps are executed
    Given an implemented step
    Then a step that isnt implemented yet
```

Reply:
Thanks - I've refactored based on your suggestion above.